src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyInterceptionValidator.cs

```diff
@@ -37,7 +37,11 @@ public bool MethodCanBeInterceptedOnInstance(MethodInfo method, object? callTarg
 #else
                 var explicitImplementation = method.Name.Contains('.');
 #endif
-                if (!explicitImplementation)

```

Comment:
I'm not sure why we allowed explicit interface implementations here before. As far as I can tell, there's no way to fake the method if we reach that point: the method is sealed, it doesn't matter if it's an explicit implementation or not (except for returning a different message).

If the fake implements the interface, we enter the first `if` and return null on line 30, so we don't reach this point.